docker-compose.yml

```diff
@@ -1,7 +1,8 @@
 version: "2.1"
 
 networks:
-  ledger:
```

Comment:
this is all redundant, if left empty all containers will simply by added to the default network which can then be renamed "ledger".